config/routes.rb

```diff
@@ -1,6 +1,9 @@
-# frozen_string_literal: true
-
 Rails.application.routes.draw do
+  root 'users#index'
+  resources :users do
+    resources :posts do
+    end
+  end
```

Comment:
- [ ] Based on the requirements, there should only be 4 routes, and we should no expose unnecessary routes. Kindly refactor the routes to only expose the index and show routes.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```